### PR TITLE
Typings for intl-locales-supported

### PR DESCRIPTION
**Pull request by Slessi**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Review by plantain-00 — APPROVED** [review]

File: types/intl-locales-supported/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Type definitions for intl-locales-supported 1.0
+// Project: https://github.com/yahoo/intl-locales-supported
+// Definitions by: Edward Sammut Alessi <https://github.com/Slessi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default function areIntlLocalesSupported(locales: string | string[]): void;
```
Comment (rbuckton) [inline types/intl-locales-supported/index.d.ts]: This is not a `default` export according to the [source](https://github.com/yahoo/intl-locales-supported/blob/master/index.js#L8). `module.exports = ` equates to `export =` in TypeScript.

File: types/intl-locales-supported/intl-locales-supported-tests.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import areIntlLocalesSupported from "intl-locales-supported";
```
Comment (rbuckton) [inline types/intl-locales-supported/intl-locales-supported-tests.ts]: This should be `import areIntLocalesSupported = require("intl-locales-supported");`

**Review by rbuckton — CHANGES_REQUESTED** [review]

File: types/intl-locales-supported/index.d.ts (hunk 1 above)
Comment (Slessi) [reply types/intl-locales-supported/index.d.ts]: done

File: types/intl-locales-supported/intl-locales-supported-tests.ts (hunk 2 above)
Comment (Slessi) [reply types/intl-locales-supported/intl-locales-supported-tests.ts]: done

**Comment by Slessi** [comment]
@rbuckton yo

**Comment by Slessi** [comment]
@rbuckton changes have been ready for 23 days, its only a two-line definition, can we approve?

**Comment by minestarks** [comment]
Here it goes, sorry for the delay!

**Comment by Slessi** [comment]
Thanks!

**Pull request merged**